src/IRCNetwork.cpp

```diff
@@ -424,6 +424,18 @@ bool CIRCNetwork::ParseConfig(CConfig* pConfig, CString& sError,
         for (const CString& sValue : vsList) {
             CString sModName = sValue.Token(0);
             CString sNotice = "Loading network module [" + sModName + "]";
+            CString sModRet;
+            CString sArgs = sValue.Token(1, true);
+
+            // We removed partyline on 2019-04-12, znc 1.8
```

Comment:
I don't think this should be here. Removing partyline is a breaking change. If user had it loaded, probably they need it, and silently making it not work anymore in that case would be bad.

Reply:
I've changed the code to explain why it failed instead, which can then be removed (if we want) in the next-next release of ZNC.  I think this resolves your concern.
```

[zarthus@nomenclature dev/znc/build.compile] master  ^?$ ./bin/znc -d /tmp/znc                                                                       [d94605c3]
[ .. ] Checking for list of available modules...
[ .. ] Opening config [/tmp/znc/configs/znc.conf]...
[ .. ] Loading global module [webadmin]...
[ .. ] Binding to port [+1111]...
[ ** ] Loading user [za]
[ ** ] Loading network [freenode]
[ .. ] Loading network module [partyline]...
[ !! ] Unable to find module partyline
[ ** ] NOTICE: [partyline] is unavailable, cannot load.
[ ** ] NOTICE: [partyline] was removed in this release of ZNC. Please either remove it from your config or install it as a third party module with the same name.
[ ** ] NOTICE: More info can be found on https://wiki.znc.in/Partyline
[ ** ]
[ ** ] Unrecoverable config error.
```